inc/block-patterns.php

```diff
@@ -12,33 +12,28 @@
 
 /**
  * Register Block Pattern Category.
- *
- * WIP
- *
+ */
 if ( function_exists( 'register_block_pattern_category' ) ) {
 
 	register_block_pattern_category(
 		'twentytwentyone',
 		array( 'label' => __( 'Twenty Twenty One', 'twentytwentyone' ) )
 	);
 }
- */
 
 /**
  * Register Block Patterns.
- *
- * WIP
- *
+ */
 if ( function_exists( 'register_block_pattern' ) ) {
 
 	register_block_pattern(
-		'',
+		'twentytwentyone/two-images-showcase',
 		array(
-			'title'      => __( '', 'twentytwentyone' ),
-			'categories' => array( 'twentytwentyone' ),
-			'content'    => '',
+			'title'         => __( 'Two Images Showcase', 'twentytwentyone' ),
+			'categories'    => array( 'twentytwentyone' ),
+			'viewportWidth' => 1440,
+			'content'       => '<!-- wp:media-text {"mediaId":1747,"mediaLink":"' . esc_url( get_template_directory_uri() ) . '/assets/images/la_mousme.jpg","mediaType":"image"} --><div class="wp-block-media-text alignwide is-stacked-on-mobile"><figure class="wp-block-media-text__media"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/la_mousme.jpg" alt="&quot;La Mousmé&quot; by Vincent Van Gogh (1888)" size-full"/></figure><div class="wp-block-media-text__content"><!-- wp:image {"align":"center","width":400,"height":512,"sizeSlug":"large","className":"is-style-twentytwentyone-image-frame"} --><figure class="wp-block-image aligncenter size-large is-resized is-style-twentytwentyone-image-frame"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/the_berceuse_woman-rocking_a_cradle.jpg" alt="&quot;The Berceuse, Woman Rocking a Cradle&quot; by Vincent Van Gogh (1889)" width="400" height="512"/></figure><!-- /wp:image --></div></div><!-- /wp:media-text -->',
```

Comment:
All text strings in the theme needs to be translation ready.

Reply:
Should be fixed in 140717d.